### PR TITLE
Update README.md to include troubleshooting for 'Undefined symbols for architecture x86_64' error

### PR DESCRIPTION
**Pull request by jamesxabregas**
Included extended troublshooting information regarding the 'Undefined symbols for architecture x86_64' error in response to issue #43.

Thank you for sending the PR! We appreciate you spending the time to work on these changes.
Help us understand your motivation by explaining why you decided to make this change.

If this PR fixes an issue, type "Fixes #issueNumber" to automatically close the issue when the PR is merged.

Test Plan:

Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work. Bonus points for screenshots and videos!

### CODE REVIEW

**Review by mikehardy — APPROVED** [review]
Nice - it's a big and very specific addition but having followed issue #43 I can't think of a better way to help devs other than to have that error chunk in there to make sure it is searchable and people connect the problem with the solution as you mention, @jamesxabregas 

+1 from me - thanks for posting this

File: README.md [hunk 1]
```diff
@@ -161,9 +161,33 @@ The `AppDelegate.m` file can only have one method for `openUrl`. If you're also
...
+    _swift_FORCE_LOAD$swiftWebKit$_FBSDKCoreKit in libFBSDKCoreKit.a(Permission.o)
+    _swift_FORCE_LOAD$swiftWebKit$_FBSDKCoreKit in libFBSDKCoreKit.a(Settings.o)
+    _swift_FORCE_LOAD$swiftWebKit$_FBSDKLoginKit in libFBSDKLoginKit.a(FBLoginButton.o)
+    _swift_FORCE_LOAD$swiftWebKit$_FBSDKLoginKit in libFBSDKLoginKit.a(LoginManager.o)
+    _swift_FORCE_LOAD$swiftWebKit$_FBSDKShareKit in libFBSDKShareKit.a(Enums+Extensions.o)
+    (maybe you meant: _swift_FORCE_LOAD$swiftWebKit$_FBSDKLoginKit, _swift_FORCE_LOAD$swiftWebKit$_FBSDKShareKit , _swift_FORCE_LOAD$swiftWebKit$_FBSDKCoreKit )
+    ld: symbol(s) not found for architecture x86_64
+```
+   
+   After **facebook-ios-sdk v7** you need to create a swift file like so ([File.Swift](https://github.com/facebook/react-native-fbsdk/blob/master/example/ios/RNFBSDKExample/File.swift)) in the main project folder. When you add an empty swift file, XCode will ask you if you want to "Create Bridging Header".
```
Comment (thebergamo) [inline README.md:184]: You could just fix the link pointing to this repo in this case

**Review by thebergamo — APPROVED** [review]
Besides the minor change I'm happy to merge this piece of docs as well! Thanks for that @jamesxabregas

File: README.md (hunk 1 above)
Comment (jamesxabregas) [reply README.md]: Updated the link.

**Pull request merged**